FirebaseDatabase/Tests/Integration/FIRDatabaseTests.m

```diff
@@ -469,6 +464,16 @@ - (void)testCallbackQueue {
   [database goOffline];
 }
 
+- (void)testSetEmulatorSettings {
```

Comment:
Also add a test showing that this doesn't throw in a normal situation and that the `ref` you get back after `useEmulator` points at the emulator.

Reply:
Done.